content/docs/command-reference/exp/save.md

```diff
@@ -0,0 +1,163 @@
+# exp save
+
+Save the current project status as a [DVC experiment].
```

Comment:
project status -> project wokspace? or project state?

Reply:
```suggestion
Save the current project workspace as a [DVC experiment].
```

The CLI help is `Save current workspace as a dvc experiment.` I think using workspace makes sense here since that's what will be captured (as opposed to .dvc file state, git staging area, etc.).